src/main/java/space/obminyashka/items_exchange/service/MailService.java

```diff
@@ -35,5 +35,5 @@ public interface MailService {
      * Reset user password by email
      * @param email for finding user by email
      */
-    void sendMessagesToEmailForResetPassword(String email);
+    UUID sendMessagesToEmailForResetPassword(String email, EmailType emailType, String host);
```

Comment:
Why don't you use the existing method for sending emails instead of the current one?

Reply:
++